pkgs/by-name/li/lime3ds/package.nix

```
@@ -0,0 +1,155 @@
+{ alsa-lib
+, boost
+, catch2_3
+, cmake
+, cryptopp
+, cpp-jwt
+, doxygen
+, enet
+, fetchzip
+, fetchurl
+, fmt
+, ffmpeg-headless
+, gamemode
+, httplib
+, inih
+, lib
+, libGL
+, libjack2
+, libpulseaudio
+, libunwind
+, libusb1
+, nlohmann_json
+, openal
+, openssl
+, pipewire
+, pkg-config
+, portaudio
+, sndio
+, spirv-tools
+, soundtouch
+, stdenv
+, vulkan-headers
+, vulkan-loader
+, xorg
+, zstd
+, enableSdl2Frontend ? true , SDL2
+, enableQt ? true , kdePackages
+, enableQtTranslations ? enableQt
+, enableCubeb ? true , cubeb
+, useDiscordRichPresence ? false , rapidjson
+}: let
+  compat-list = fetchurl {
+    name = "lime3ds-compat-list";
+    url = "https://raw.githubusercontent.com/Lime3DS/compatibility-list/master/compatibility_list.json";
```

```suggestion
    url = "https://raw.githubusercontent.com/Lime3DS/compatibility-list/b0c8b6b80d716db6b957ba103c7a9e17ead24d55/compatibility_list.json";
```
otherwise the hash doesn't match anymore with the next commit

---

Thanks!